src/components/Modal/IdentityConnectModal/ForumConnect.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+
+import useFormatMessage from '../../../hooks/useFormatMessage.ts'
+import useIsProfileValidated from '../../../hooks/useIsProfileValidated.ts'
+import { AccountType } from '../../../types/users.ts'
+import ActionCard from '../../ActionCard/ActionCard.tsx'
+import CircledForum from '../../Icon/CircledForum.tsx'
+
+import { FlowType } from './AccountsConnectModal.tsx'
+import ForumConnectionFlow from './ForumConnectionFlow.tsx'
+
+export interface FlowProps {
```

Comment:
Shouldn't this be in `AccountsConnectModal` along with FlowType ?

Reply:
This is the props interface of the three `Connect` components, I renamed it to `ConnectProps` in the pr that follows, wdyt?